src/ContextMenu.tsx

```diff
@@ -70,13 +72,41 @@ const ContextMenuView: React.FunctionComponent<IContextMenuView> = props => {
   const { menuItems = [], ...viewProps } = props;
 
   const { contextMenu: theme } = useTheme();
+  const [isVisible, setIsVisible] = React.useState(false);
 
   return (
-    <Box {...viewProps} as={ReactContextMenu} css={menuStyles(theme, menuItems.length > 0)}>
-      {menuItems.map((item, index) => {
-        return <ContextMenuItem key={index} {...item} />;
-      })}
-    </Box>
+    <>
+      {props.blockExternalClicks && isVisible
+        ? ReactDOM.createPortal(
```

Comment:
Suggest using Portal component from UI-Kit.
```suggestion
     <Portal>
```